.github/workflows/tests.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.7, 3.11]
+        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
```

Comment:
Why not just test 3.9 and 3.13? Is there risk that the versions in-between would fail while the extremes wouldn't?

Supporting any Python version in tox makes sense for flexible local testing, but maybe not for CI, which is run on every change

Reply:
> Is there risk that the versions in-between would fail while the extremes wouldn't?

It rarely happens, but yes it it possible, e.g.
- There are bugfix releases for existing Pythons, unfortunately sometimes they may even [add API](https://github.com/pydantic/pydantic/pull/2306)
- All sorts of compatibility code such as https://github.com/executablebooks/mdformat/blob/master/src/mdformat/_compat.py means there can be combinations that only break in the middle of the matrix

And, GitHub Actions is free, so why not? :)